internal/artifacts/module_arch.go

```diff
@@ -138,17 +180,22 @@ func collectSelectedModulesAndDependencies(mtaObj *mta.MTA, modulesWithDependenc
 }
 
 func buildModules(source, target string, extensions []string, modulesToBuild []string,
-	modulesToPack map[string]bool, wdGetter func() (string, error)) error {
+	modulesToPack map[string]bool, wdGetter func() (string, error)) (map[string]string, error) {
```

Comment:
Please name the new output parameter (to something like `packedModulePaths`)

Reply:
done